RealmSwift/Realm.swift

```diff
@@ -391,13 +391,13 @@ public final class Realm {
 
     - warning: This method is useful only in specialized circumstances.
 
-    - parameter className: The class name of the objects to be returned.
+    - parameter typeName: The class name of the objects to be returned.
 
     - returns: All objects for the given class name as dynamic objects
 
     :nodoc:
     */
-    public func allDynamicObjects(ofType typeName: String) -> Results<DynamicObject> {
+    public func dynamicObjects(_ typeName: String) -> Results<DynamicObject> {
```

Comment:
Same here; something like `objects(ofTypeName:)` or hopefully something more consistent with the above suggestion. It seems like our dynamic object API is weirdly distant from out non-dynamic API.